sitemedia/js/controllers/search_controller.js

```diff
@@ -1,17 +1,20 @@
 // src/controllers/search.js
 
 import { Controller } from "@hotwired/stimulus";
-import { ApplicationController, useDebounce } from "stimulus-use";
+// TODO: Re-enable debounce when Turbo is set up
+// import { ApplicationController, useDebounce } from "stimulus-use";
 
 export default class extends Controller {
     static targets = ["query", "sort", "sortlabel"];
-    static debounces = ["submit"];
+    // static debounces = ["submit"];
 
     connect() {
-        useDebounce(this);
+        // useDebounce(this);
     }
 
     submit() {
+        // Close modal if open before submitting form
+        window.location.href = "#";
         this.element.submit();
     }
```

Comment:
@rlskoeser Can you test again to confirm it's working now, and is this an OK way to handle closing it? (I couldn't figure out how else to consistently close the modal without breaking the strict coupling of the anchor in the URL and visibility…)